src/components/MultipleDatePicker.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import Button from './Button';
+import DayPicker from 'react-day-picker/DayPicker';
+import 'react-day-picker/lib/style.css';
+import styled from 'styled-components';
+import { observable, action } from 'mobx';
+import { observer } from 'mobx-react';
+import { Calendar } from './Icons';
+
+@observer
+class MultipleDatePicker extends Component {
+  @observable show = false
+  @observable current = new Date()
+
+  @action.bound YearMonthForm = ({ date, localeUtils }) => {
```

Comment:
This doesn't need the `@action.bound` decorator. The action decorator is used to tell mobx that a function might change observable values. The `handleChange` function for example should probably be an action.